### PR TITLE
Tweaks instagib lasers

### PR DESCRIPTION
**Pull request by Kyep**
Instagib lasers now work on non-carbon mobs (such as simple_animals).

🆑 Kyep
fix: Instagib lasers now work against non-carbon mobs, such as animals and borgs.
/🆑

### CODE REVIEW

File: code/modules/projectiles/projectile/beams.dm [hunk 1]
```diff
@@ -130,7 +130,9 @@
...
-	if(iscarbon(target))
-		var/mob/living/carbon/M = target
-		M.visible_message("<span class='danger'>[M] explodes into a shower of gibs!</span>")
-		M.gib()
+	if(isliving(target))
+		var/mob/living/L = target
+		L.visible_message("<span class='danger'>[L] explodes!</span>")
+		L.gib()
+	else if(isturf(target) || istype(target,/obj/structure/))
+		target.ex_act(2)
```
Comment (Crazylemon64) [inline code/modules/projectiles/projectile/beams.dm]: `ex_act(1)` is the most powerful explosion. Or did you want something more moderate?

**Comment by FlattestGuitar** [comment]
Thank God.

Finally I can gib all those crabs.

File: code/modules/projectiles/projectile/beams.dm (hunk 1 above)
Comment (Kyep) [inline code/modules/projectiles/projectile/beams.dm]: Pulse beams already use ex_act(2). I figure this shouldn't be weaker than pulse.

**Comment by Fox-McCloud** [comment]
Let's not make a rifle meant for high stakes mob vs mob combat into a rifle that can tear literally anything/everything appart.

👍 on the `living` change, but 👎 on the other.

**Comment by Kyep** [comment]
@Fox-McCloud Changed

**Comment by Fox-McCloud** [comment]
Needs a CL, please.

**Pull request merged**